index.js

```diff
@@ -34,6 +34,10 @@ var Dropzone = React.createClass({
     accept: React.PropTypes.string,
   },
 
+  componentDidMount: function() {
+    this._enterCnt = 0;
```

Comment:
I would not save bytes here and go for explicit naming: `enterCounter` or similar. Let minifier take care of byte shaving...